app/_src/kubernetes-ingress-controller/guides/services/grpc.md

```diff
@@ -16,7 +16,12 @@ For this example, you need to:
 
 To make `gRPC` requests, you need a client that can invoke gRPC requests. You can use [`grpcurl`](https://github.com/fullstorydev/grpcurl#installation) as the client. Ensure that you have it installed on your local system.
 
-{% include_cached /md/kic/prerequisites.md release=page.release disable_gateway_api=false gateway_api_experimental=true %}
+{% assign grpcroute_is_experimental = false %}
+{% if_version lte:3.1.x %}
```

Comment:
This is somewhat tricky to get 100% right because the GRPCRoute being in alpha or not (or rather GA or not) is dependant on the Gateway API version and only directly on the KIC version.

To solve this I'd suggest to also include a conditional in https://github.com/Kong/docs.konghq.com/blob/main/spec/fixtures/app/_includes/md/kic/prerequisites.md#L17-L18: make Gateway API version depending on the KIC version there.

WDYT?